"Uninstall libraries from Arduino IDE" updated for 2.0.0 (#134)
arduino/help-center-content: content/Software Support/IDE Settings/Uninstall-libraries-from-Arduino-IDE.md

---

<a id="library-manager"></a>

## Uninstall libraries with the Library manager

The Library Manager in Arduino IDE 2 can be used to uninstall individual libraries:

1. Click the ![Library Manager button](img/symbol_library.png) **Library Manager** button or select _Tools > Manage Libraries..._ in the menu bar.

   ![Opening the Library Manager.](img/ide2-library-manager-open.png)

2. Filter your search:

   * For the Type option, select **Installed** to show only installed libraries
   * To further filter the results, type the library name into the text field.

   ![Filtering the results.](img/ide2-library-manager-filter.png)

3. Find the library in the search results. Note that the results are listed alphabetically.

4. Put your mouse cursor over the INSTALLED label and click UNINSTALL to uninstall the library.

   ![Uninstalling the ArduinoBLE library.](img/ide2-library-manager-uninstall.png)

5. Confirm that you want to uninstall the library by clicking **Yes**.

---

<a id="file-manager"></a>

## Remove libraries with your file manager

> Removing the library files directly is the most reliable method that will work for versions of Arduino IDE and library types.

Author: ```suggestion
This method allows you to remove many libraries at once. It works for all types of libraries and versions of Arduino IDE.
```
